Week-14/Link_List_1/Remove_Duplicates.js

```diff
@@ -0,0 +1,17 @@
+function removeDuplicates(head) {
```

Comment:
[mandatory]: declare the linkedlist class

Reply:
declared